### DISCUSSION TITLE
Dragging images to a specific folder(Scanning an external folder)

### DISCUSSION THREAD

**Discussion by yossiyemin** [immich-app/immich#5529] [General]
Hello,
I installed the system to docker based on windows.
Everything works fine.
I have a lot of pictures saved on a certain drive that I want to be displayed in immich.
How can you make the machine scan all the images that are in an external folder?

**Comment by bo0tzz** [comment]
https://immich.app/docs/features/libraries

**Reply by yossiyemin** [reply]
Thank you very much, do you know maybe what I need to write in the path to point to the folder in windows?
